x/mint/types/constants.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	SecondsPerMinute = 60
+	MinutesPerHour   = 60
+	HoursPerDay      = 24
+	// DaysPerYear is the mean length of the Gregorian calendar year. Note this
+	// value isn't 365 because 97 out of 400 years are leap years. See
+	// https://en.wikipedia.org/wiki/Year
```

Comment:
https://www.britannica.com/story/ten-days-that-vanished-the-switch-to-the-gregorian-calendar If you're curious about how the transition went